config/ant-phase-verify.xml

```diff
@@ -119,6 +119,10 @@
           <!-- Exclude because printed AST has line length > 100 characters -->
           <exclude name="**/InputJava14InstanceofWithPatternMatchingAST.txt"/>
           <exclude name="**/InputJava14Records.txt"/>
+          <exclude name="**/InputJava14TextBlocks.txt"/>
+
+          <!-- line length must be over 100 chars to properly suppress warnings -->
+          <exclude name="**/intellij-idea-inspections.xml"/>
```

Comment:
this worries me .... we did a lot to keep this file on leash 

lets change name of inner class to `AssertGenTextBlockLexer` or `AssertGeneratedTextLexer` something similar to avoid this suppression.

Reply:
Fixed.